packages/mdc-segmented-button/segment/component.ts

```diff
@@ -21,43 +21,96 @@
  * THE SOFTWARE.
  */
 
-import {MDCComponent} from '@material/base/component';
+import {MDCComponent} from '../../mdc-base/component';
```

Comment:
I change this to develop/test locally, and occasionally I forget to change it back.